src/clj/rems/db/applications.clj

```diff
@@ -37,65 +37,28 @@
 
 ;;; Creating applications
 
-(defn allocate-external-id! [prefix]
+(defn- allocate-external-id! [prefix]
   (conman/with-transaction [rems.db.core/*db* {:isolation :serializable}]
     (let [all (db/get-external-ids {:prefix prefix})
           last (apply max (cons 0 (map (comp read-string :suffix) all)))
           new (str (inc last))]
       (db/add-external-id! {:prefix prefix :suffix new})
       {:prefix prefix :suffix new})))
 
-(defn format-external-id [{:keys [prefix suffix]}]
+(defn- format-external-id [{:keys [prefix suffix]}]
   (str prefix "/" suffix))
 
 (defn application-external-id! [time]
```

Comment:
private?

Reply:
It's tested directly from `rems.db.test-applications/test-application-external-id!`

Reply:
It's just kinda weird that allocate-application-ids! is private and this isn't. not a big thing but you could consider making this private and stealing it with `@#'` in the test.